src/components/EventForm/formParts/RadioGroupButtons2.js

```diff
@@ -0,0 +1,108 @@
+import React, { useState, component } from "react"
+import { Button, TextField } from "@material-ui/core";
+import Radio from '@material-ui/core/Radio';
+import RadioGroup from '@material-ui/core/RadioGroup';
+
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import FormLabel from '@material-ui/core/FormLabel';
+import Checkbox from '@material-ui/core/Checkbox';
+
+
+export default function Form(props) {
+  
+  const [otherButtDisabled, setOtherButtDisabled] = useState(true)
+
+  const radioGroupLabelFontSize = '17px'
+
+  const {
+      values: { name, email, password, confirmPassword, id, firstname, lastname, other_option },
+      errors,
+      touched,
+      handleSubmit,
+      handleChange,
+      isValid,
+      setFieldTouched, 
+      groupName, optionsArray, otherOptionRequired
+  } = props;
+
+  const change = (name, e) => {
+      e.persist();
+      handleChange(e);
+      setFieldTouched(name, true, false);
+  };
+
+  console.log(props);
+  console.log(groupName);
+  console.log(optionsArray);
+  console.log(otherOptionRequired);
+  
+  // let otherOptionRequired = true;
+  // let optionsArray = ["1st Year","2nd Year","3rd Year","4th Year","5+ Yearrhae -  FROM DA COMP"];
+  // let groupName = "FROM THE COMP";
+  
+  // console.log(props);
+  // console.log(groupName);
+  // console.log(optionsArray);
+  // console.log(otherOptionRequired);
+
+  return (
+    <React.Fragment>
+      <FormLabel component="legend" style={{fontSize: radioGroupLabelFontSize}}>{groupName}</FormLabel>  
```

Comment:
What is legend?

Reply:
Not sure but nothing changed from what I can tell when removed, so I removed it.